data/Greenfunctions/2A2/GU3.jl

```diff
@@ -1,5 +1,5 @@
 using ..GenericCharacterTables
-import ..GenericCharacterTables: ExtendableMatrix, SimpleCharTable
+import ..GenericCharacterTables: SimpleCharTable
```

Comment:
This PR is getting a bit unwieldy. Here is a suggestion so that we can merge parts of it quickly now, and reduce this PR to its core: namely, let's put the changes which expunge `ExtendableMatrix` from the `data` files (without changing anything else) into their own (initial) commit of this PR, and then also open another PR with just that commit.

We can work on that together on Wednesday if you like, but you can also try it out now. It involves some mild `git` history rewriting. Let me sketch what I would do below (apologies if you already know all of this):
1. First make sure there are no uncommitted changes; also perhaps do a `git branch backup` to keep a pointer to your top commit just in case something goes wrong
2. now go to the root directory of your repository clone and do `git reset master` (this will remove all commits but keep your changes)
3. `git add data src/Convert/Convert.jl && git commit -m "Remove ExtendableMatrix references from data/"`
4. `git commit -m "add AbstractGenericCharacter type" src test`  -- commit rest of the changes
5. `git checkout -b rm-ExtendableMatrix`   -- switch to new branch
6. `git reset --hard HEAD^`  -- discard the last commit (it's still safely on your original branch)
7. edit `src/CharTable/jl` to add a `CharTable` constructor just like you do in this PR, only it just passes all arguments through, except for the second argument `table`, which is turned into `ExtendableMatrix(table)`
8. `git commit -m "Adjust CharTable constructor"`
9. create a pull request (`gh pr create` if you use the `gh` command line tool)
10. afterwards switch back to the original branch and rebase that onto the new one to simplify things: `git rebase rm-ExtendableMatrix` (depending on how and where you added the new constructor in step 7, this will either require resolving a merge conflict, or removing a now duplicate `CharTable` constructor.

Reply:
(err, and obviously you also need to treat `SimpleCharTable` the same way by adding an auxiliary constructor for it, too

Reply:
I hope I got you right, but I think this should be done now.